### PR TITLE
DOC-5663 log export log name node number

### PR DESCRIPTION
**Pull request by andf-crl**
Addresses: DOC-5663

- Added new Log name format section, showing naming format for cloud sink-written logs
- Added related FAQ entry.

Question: Is `{region}` the _source_ region of the log-writing-CC:D node, as I have indicated here? Or is there another factor I am missing? Thanks!

[export-logs.md](https://deploy-preview-15103--cockroachdb-docs.netlify.app/docs/cockroachcloud/export-logs#log-name-format)

### CODE REVIEW

**Review by dhartunian — APPROVED** [review]
:lgtm: 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/15103#-:-NJkQAVZ7WX8YOHNv1Md:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained

File: cockroachcloud/export-logs.md [hunk 1]
```diff
@@ -295,6 +311,10 @@ Currently, the following CockroachDB [log channels](/docs/{{site.versions["stabl
 
 Yes, the [SQL Audit Log](/docs/{{site.versions["stable"]}}/sql-audit-logging.html) is exported via the `SENSITIVE_ACCESS` log channel by default, as long as you have previously enabled audit logging on desired tables using the [`ALTER TABLE ...EXPERIMENTAL_AUDIT`](/docs/{{site.versions["stable"]}}/experimental-audit.html) statement.
 
+### Why are some logs appearing without a node number in the name?
+
+Log messages received from {{ site.data.products.dedicated }} nodes that are not yet fully started may arrive without a node number appended to the log name, in the format `{logname}.n`. Node-specific log messages, as they are received, are written to node-specific logs in the format `{logname}.n1`, `{logname}.n2`, etc. where the number following the `n` characters is the node ID. See [Log Name Format](#log-name-format).
```
Comment (MichaelTrestman) [inline cockroachcloud/export-logs.md:316]:
```suggestion
Log messages received from {{ site.data.products.dedicated }} nodes that are not yet fully started may arrive without a node number appended to the log name, in the format `{logname}.n`. Node-specific log messages, as they are received, are written to node-specific logs in the format `{logname}.n1`, `{logname}.n2`, etc., where the number following the `n` characters is the node ID. See [Log Name Format](#log-name-format).
```

**Review by MichaelTrestman — APPROVED** [review]
LGTM!

**Review by dhartunian — APPROVED** [review]
LGTM

**Pull request merged**